### ISSUE TITLE
On status FAILED continues to run indefinitely

### ISSUE THREAD

**Issue by soile1991** [GitHub30/gdrive.sh#5]
"status": "FAILED",
 "percentDone": 100,
  "status": "FAILED",
 "percentDone": 100,
  "status": "FAILED",
  "percentDone": 100,
  "status": "FAILED",
  "percentDone": 100,
  "status": "FAILED",

**Comment by GitHub30** [comment]
I tried following command for downloading folder. But it works.

```bash
curl gdrive.sh | bash -s 1TdFtOzKlGQnckOavxlwgJuW4nc9oGU6U
```

https://colab.research.google.com/drive/1k5tOZs4wHEMEZOUUlE-LeGRT_pEom7eF

https://travis-ci.org/GitHub30/gdrive.sh

Please tell me your executed command.

**Comment by soile1991** [comment]
Yes, it works very well. The problem is if for some reason you get "status": "FAILED". The script runs indefinitely and never stops it keeps printing.

`"status": "FAILED",
"percentDone": 100,
"status": "FAILED",
"percentDone": 100,
" `

**Comment by GitHub30** [comment]
Thank you. I will fix it. If response contains FAILED, exit 1.
